pkg/controller/cincinnati/new.go

```diff
@@ -107,6 +134,9 @@ func newKubeResources(instance *cv1alpha1.Cincinnati, image string) (*kubeResour
 	k.envConfigHash = envConfigHash
 	k.podDisruptionBudget = k.newPodDisruptionBudget(instance)
 	k.graphBuilderContainer = k.newGraphBuilderContainer(instance, image)
+	if k.UseInitContainer(instance) {
```

Comment:
I think you could update `newGraphDataInitContainer` to simply return `nil` if the `.Spec.GraphDataImage == ""` and update this to be:

```suggestion
	k.graphDataInitContainer = k.newGraphDataInitContainer(instance)
```